src/test/java/com/iexec/sms/tee/session/gramine/GramineSessionMakerServiceTests.java

```diff
@@ -46,13 +45,12 @@ void beforeEach() {
     // region getSessionYml
     @Test
     void shouldGetSessionJson() throws Exception {
-        TeeEnclaveConfiguration enclaveConfig = mock(TeeEnclaveConfiguration.class);
+        TeeEnclaveConfiguration enclaveConfig = TeeEnclaveConfiguration.builder()
+                .fingerprint(APP_FINGERPRINT).entrypoint("/apploader.sh").build();
```

Comment:
I find it clearer on multiple lines.
```suggestion
        TeeEnclaveConfiguration enclaveConfig = TeeEnclaveConfiguration.builder()
                .fingerprint(APP_FINGERPRINT)
                .entrypoint("/apploader.sh")
                .build();
```

Reply:
done in c1d5287